### PR TITLE
Apply some eslint fixes

### PR DESCRIPTION
**Pull request by Holzhaus**
This fixes eslint warnings for Mixxx' JS files (that are not controller mappings) and the Roland DJ-505 mapping.

I'm aware that the coding guidelines state:
> Do not send us patches that are purely cosmetic with respect to source changes – this is a waste of time since it does not benefit users directly.

However, this also fixes an undeclared variable in `midi-components` (I'd consider this a bug) and I hope this is okay this time.

### CODE REVIEW

**Comment by uklotzde** [comment]
LGTM.

@Be-ing?

File: script/svg/__init__.js [hunk 1]
```diff
@@ -4,64 +4,59 @@ __setupPackage__(__extension__);
...
+    svg.templateHooks.variable = function(varName){
+        if (varName in global){
             return global[varName];
         }
-        return '';
-    }
+        return "";
+    };
+
+    svg.templateHooks.prop = function(propName, varName){
```
Comment (uklotzde) [inline script/svg/__init__.js]: Doesn't the new config detect and point out the missing space between adjacent braces?

Comment (Holzhaus) [reply script/svg/__init__.js]: No, I forgot to add that rule:
https://eslint.org/docs/rules/space-before-blocks

I'll make a new PR and add it.

Comment (Holzhaus) [reply script/svg/__init__.js]: Here it is: #2457

**Comment by uklotzde** [comment]
LGTM. Ready to merge?

**Comment by Holzhaus** [comment]
Yup.

**Pull request merged**